### PR TITLE
Add |JacksonAllInObjectScope| to add all Embulk columns as a JSON Object node.

### PR DESCRIPTION
**Pull request by dmikurube**
@sakama Though it's still WIP, it will be the class to export all Embulk columns into one JSON object node without specifying column names. Can you have a quick look?

### CODE REVIEW

File: src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    public ObjectNode scopeObject(final SinglePageRecordReader singlePageRecordReader)
+    {
+        final ObjectNode resultObject = JsonNodeFactory.instance.objectNode();
+
+        singlePageRecordReader.getSchema().visitColumns(new ColumnVisitor()
+            {
+                @Override
+                public void booleanColumn(Column column)
+                {
+                    resultObject.put(column.getName(),
```
Comment (sakama) [inline src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java:41]: `.put()` was deprecated.
`.set()` is better?

File: src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                                     new TextNode(singlePageRecordReader.getString(column)));
+                }
+
+                @Override
+                public void timestampColumn(Column column)
+                {
+                    if (timestampFormat == null) {
+                        resultObject.put(column.getName(),
+                                         new LongNode(singlePageRecordReader.getTimestamp(column).getEpochSecond()));
```
Comment (sakama) [inline src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java]: It's helpful if also support milli sec and nano sec convert.

**Review by sakama — COMMENTED** [review]
I tested with Elasticearch output. And works fine with long, double, string, time(second).

File: src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java (hunk 1 above)
Comment (dmikurube) [reply src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java:41]: Ah, that's right here. I may use `put(String, boolean)` instead.

File: src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java (hunk 2 above)
Comment (dmikurube) [reply src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java]: Yeah, that'd be a design choice. These `Scope` classes are easily implemented in each plugin. Then I thought "standard" classes (contained in the library package) to keep simple and minimum as supporting all requirements within the library can need many options, many configurations, many classes, …

Do you have ideas how we can support milli/nano sec while we keep the class interface (constructor) simple?

**Review by dmikurube — COMMENTED** [review]
Thanks for the comments!

File: src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java (hunk 2 above)
Comment (dmikurube) [reply src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java]: Finally, I thought millisec and nanosec could be covered by timestamp formatting, such as `%s.%3N` and `%s.%6N`. What do you think?

It is a special case which stores the epoch as `long` while the other case stores as `String`. The solution above does not support millisecs in `long`, but I don't think it's very popular...

**Comment by dmikurube** [comment]
@sakama Updated `JacksonAllInObjectScope` to work for all types. Can you take a look?

**Comment by sakama** [comment]
@dmikurube Sorry for delaying. I'll try it.

**Comment by dmikurube** [comment]
@sakama If it helps you to release a new version to try, I'll merge this and #70 after reviewing.

**Review by sakama — APPROVED** [review]
Got it.
Let's go ahead! thanks.

**Comment by dmikurube** [comment]
Thanks! Merging and releasing.

**Pull request merged**